### PR TITLE
time slice alerting support

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
**Not yet ready for merge or review**

To-do:
- Update time slice page with info about 1 vs 5 min granularity and config

### What does this PR do? What is the motivation?

Doc updates preparing for Time slice GA release at end of quarter

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by roxanne-moslehi** [comment]
@estherk15 Another SLO doc update that is ready to review and finalize, but we will need to wait for the release happening in the next few days before merging it in

CC @zhengshizhao

**Review by estherk15 — APPROVED** [review]
Looks good, let me know when you want to merge!

**Comment by roxanne-moslehi** [comment]
@estherk15 we are good to merge this!

FYI @zhengshizhao

**Pull request merged**
